schemas/2017-11-15-privatepreview/Microsoft.DataMigration.json

```diff
@@ -775,14 +775,6 @@
     "DataMigrationServiceProperties": {
       "type": "object",
       "properties": {
-        "publicKey": {
-          "type": "string",
-          "description": "The public key of the service, used to encrypt secrets sent to the service"
-        },
-        "virtualSubnetId": {
-          "type": "string",
-          "description": "The ID of the Microsoft.Network/virtualNetworks/subnets resource to which the service should be joined"
-        }
       },
       "required": [
         "virtualSubnetId"
```

Comment:
>virtualSubnetId [](start = 9, length = 15)

property is still in required list.

Reply:
Fixed, putting the pubkey back as placeholder